nixos/modules/services/web-apps/lanraragi.nix

```
@@ -22,6 +22,10 @@ in
         description = "Port for LANraragi's web interface.";
       };
 
+      openFirewall = lib.mkEnableOption // {
```

```suggestion
      openFirewall = lib.mkEnableOption "" // {
```
oops, found a mistake